lib/hdf.py

```diff
@@ -0,0 +1,24 @@
+import h5py
+import numpy as np
+from typing import Dict
+
+
+def get_input_dict_from_returnn_hdf(hdf_file: h5py.File) -> Dict[str, np.ndarray]:
```

Comment:
Does is make sense to add the option for a Path Object?

Reply:
As this needs physical access to the underlaying file, this would anyway only be called in the worker, where we may have resolved all Path objects. I don't think it is that necessary.

Also all other library functions act directly on the file and not path objects.